antenna/util.py

```diff
@@ -139,21 +139,17 @@ def __init__(self, msg, ret):
         self.return_value = ret
 
 
-# Tuple of retry times in seconds in order of attempt.
-RETRY_TIMES = (
-    1,
-    5,
-    10,
-    30,
-    60,
-    2 * 60,
-)
-RETRY_TIMES_MAX_INDEX = len(RETRY_TIMES) - 1
-
-
-def retry(retryable_exceptions=Exception, retryable_return=None, max_attempts=10,
-          sleep_function=time.sleep, module_logger=None):
-    """Decorator for retrying with exponential wait and logging
+def wait_time_generator():
+    for amt in [1, 1, 5, 10, 30]:
+        yield amt
```

Comment:
This is a much nicer construct for dealing with "what's the next wait time" as well as "have I exceeded the maximum number of retry attempts". Previously, the retry attempts were hard-coded and we'd pass in a `max_attempts` argument. That felt like a clumsier API.